src/components/Prescription/Prescription.css

```diff
@@ -1,12 +1,21 @@
 .script-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-top: 22vh;
     border: black solid 3px;
     background-color: white;
-    height: 88%;
-    width: 95%; 
+    height: 70vh;
+    width: 40vw; 
     border-radius: 10px;
+}
+
+.top-container {
     display: flex;
-    align-items: flex-start;
-    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    height: 50%;
 }
```

Comment:
The containers in this component got moved around to facilitate this layout, just an fyi.